mainframer.sh

```diff
@@ -40,32 +61,21 @@ if [ -z "$BUILD_COMMAND" ]; then
 	exit 1
 fi
 
-pushd "$PROJECT_DIR"
+# Read exclude rules.
+LOCAL_EXCLUDE="$(readExcludesFile .mainframerignorelocal)"
+REMOTE_EXCLUDE="$(readExcludesFile .mainframerignoreremote)"
+
 # Sync project to remote machine.
-rsync --archive --delete --compress-level=$LOCAL_COMPRESS_LEVEL \
---exclude='.gradle' \
---exclude='.idea' \
---exclude='**/.git/' \
---exclude='artifacts' \
---exclude='captures' \
---exclude='**/build' \
---exclude='**/local.properties' \
---rsh "ssh" ./ "$REMOTE_BUILD_MACHINE:~/$PROJECT_DIR_NAME"
+eval "rsync --archive --delete --compress-level=$LOCAL_COMPRESS_LEVEL $LOCAL_EXCLUDE --rsh ssh ./ $REMOTE_BUILD_MACHINE:~/$PROJECT_DIR_NAME"
```

Comment:
At this point we can make this a function with four arguments. Not sure if it’s worth it.